WebRoot/js/zimbraMail/mail/ZmMailApp.js

```diff
@@ -1657,7 +1657,8 @@ function(params, callback) {
 
     if (appCtxt.isExternalAccount()) {
         var loadCallback = this._handleLoadLaunch.bind(this, params, callback);
-	    AjxDispatcher.require(["MailCore", "Mail", "Startup2"], true, loadCallback, null, true);
+	    AjxDispatcher.require(["MailCore", "Mail"], true, loadCallback, null, true);
+	    AjxDispatcher.require("Startup2", false, loadCallback, null, true);
```

Comment:
Is it okay to call them in different sync/async mode? The former is called with `async true` and the latter with `async false`. There might be a case that Startup2 is loaded faster than the others for some reason. Do they need to be loaded in the specific order; MailCore, Mail and then Startup2?

In addtion, if the issue depends on the order (or timing) of loading modules and other modules like calendar need to be loaded after the Startup2, does `async false` need to be specified on both so that MailCore, Mail and Startup2 are surely loaded here?

Reply:
Yes, in fact what I find out during debugging that the Startup2 should be loaded earlier because it has multiple utility functions which are being used during the page load. In case of regular user login scenario, it is being loaded synchronously. However for the external user it was being loaded asynchronously. And I didnt find any need of loading "MailCore" and "Mail" packages asynchronously because they are not needed during the page load. Hence I have kept them as it is.

Reply:
I understood. Thanks Sitaram.